django_cte/query.py

```diff
@@ -65,7 +67,7 @@ def chain(self, klass=None):
 
 class CTECompiler(object):
 
-    TEMPLATE = "{name} AS ({query})"
+    TEMPLATE = "{name} AS {materialized} ({query})"
```

Comment:
Perhaps a bit nitpicky, but I don't love that this will add an extra space to the query when `materialized` is an empty string (which is the default).

Consider having `get_materialized_keyword()` return `MATERIALIZED ` (with trailing space) and remove the trailing space from this template.

Reply:
`MATERIALIZED ` with trailing space seems dangerous solution for further modification. It can cause some troubles. So I decided to create `get_cte_query_template` method. It looks better and not so tricky.